lldb/source/Plugins/TypeSystem/Swift/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+add_lldb_library(lldbPluginTypeSystemSwift PLUGIN
+  TypeSystemSwift.cpp
+  TypeSystemSwiftTypeRef.cpp
+  SwiftASTContext.cpp
+
+  LINK_LIBS
+    lldbCore
+    lldbSymbol
+    lldbTarget
+    lldbUtility
+#    swiftAST
```

Comment:
Can we go ahead and delete these?

Reply:
> Would it be possible to split the pure code-movement and getTypeInfo changes into separate PRs?

Sure! I already landed that patch with a separate PR, I just didn't want to wait another 3 hours before putting this one up and it was a dependency.